File: src/main/java/net/finance/utils/DateUtil.java
package net.finance.utils;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.ZoneId;
import java.util.Date;

public class DateUtil {

    public static Date convertLocalDateTimeToDate(LocalDateTime dateTime) {
        return Date.from(dateTime.atZone(ZoneId.systemDefault()).toInstant());
    }
}

File: src/main/kotlin/com/finances/utils/DateUtil.kt
package com.finances.utils

import java.time.LocalDateTime
import java.time.ZoneId
import java.util.Date

object DateUtil {

    fun convertLocalDateTimeToDate(dateTime: LocalDateTime): Date {
        return Date.from(dateTime.atZone(ZoneId.systemDefault()).toInstant())
    }
}
